Clarify Affiliation with PolkaProject (#870)
w3f/polkadot-wiki: docs/learn-accounts.md

Alternatively, you might find some wallets on the
[Polkaproject.com page](http://www.polkaproject.com/) but bear in mind that these are **unaudited**
and not officially affiliated with Web3 Foundation or the Polkadot project unless otherwise stated.

> Keep in mind that the Web3 Foundation is also not affiliated with PolkaProject.

Reviewer: ```suggestion
PolkaProject is an independent site which is not affiliated with Web3 Foundation.
```

Clarifying this a bit here.  Also we don't use a definite article in front of W3F

Author: @laboon This change looks great to me!
